src/tests/test_claim_detector.py

```
@@ -0,0 +1,400 @@
+        
+        # 2. Check that the texts of the deduplicated claims match the expected texts (ignoring order for this check).
+        deduplicated_claim_texts = {claim.text for claim in deduplicated}
+        assert deduplicated_claim_texts == set(expected_texts_ordered)
+
+        # 3. Check that the kept claims are indeed sorted by check_worthiness (descending).
+        for i in range(len(deduplicated) - 1):
+            assert deduplicated[i].check_worthiness >= deduplicated[i+1].check_worthiness
+
+        # 4. For each expected text, ensure the version kept was the highest-scoring one from the input.
+        for expected_text in expected_texts_ordered:
+            original_versions = [c for c in claims_in if c.text == expected_text or 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)